gmaps/job.go

```diff
@@ -33,11 +33,19 @@ func NewGmapJob(id, langCode, query string, maxDepth int, extractEmail bool) *Gm
 		id = uuid.New().String()
 	}
 
+	mapURL := fmt.Sprintf("https://www.google.com/maps/search/%s", query)
+	if geoCoordinates != "" {
+		mapURL = fmt.Sprintf("https://www.google.com/maps/search/%s/@%s", query, strings.ReplaceAll(geoCoordinates, " ", ""))
+	}
+	if zoom > 0 {
```

Comment:
linter fails here 

```
mapURL := fmt.Sprintf("https://www.google.com/maps/search/%s", query)

if geoCoordinates != "" {
		mapURL = fmt.Sprintf("https://www.google.com/maps/search/%s/@%s", query, strings.ReplaceAll(geoCoordinates, " ", ""))
	}

if zoom > 0 {
    mapURL = fmt.Sprintf("%s,%dz", mapURL, zoom)
}

```

PS: you can use `make lint` before commiting. 

Thank you

Reply:
I see